### PR TITLE
Update db commands to support different user/host

### PR DESCRIPTION
**Pull request by rmarianski**
These changes allow me to fully setup the database on a different host with a different username.

### CODE REVIEW

File: scripts/setup_and_run_tests.sh [hunk 1]
```diff
@@ -53,7 +63,7 @@ function cleanup {
       kill -HUP "${server_pid}"
    fi
    echo "=== Dropping database \"${dbname}\" and cleaning up..."
-   dropdb --if-exists "${dbname}"
+   dropdb --if-exists $dbhostargs "${dbname}"
```
Comment (zerebubuth) [inline scripts/setup_and_run_tests.sh]: Does this need `$userargs` as well?

Comment (rmarianski) [inline scripts/setup_and_run_tests.sh]: Good catch! Updated in 5f65ddc.

**Comment by rmarianski** [comment]
> A small thought which might be worth looking at is whether $DBHOST and $USER might be copied, if present, into the $PGHOST and $PGUSER environment variables. That won't affect osm2pgsql, but the PostgreSQL internal tools like createdb, dropdb and psql all look at those variables.

Should we just change the variables that we look at to be the `PGHOST` and `PGUSER` variables?

**Comment by zerebubuth** [comment]
Yup, that would work! Although we'd still need to shoehorn those into the command line arguments that `osm2pgsql` is expecting.

**Pull request merged**